src/cbmc_starter_kit/template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -200,12 +209,15 @@ def get_proof_dirs(proof_root, proof_list, marker_file):
         sys.exit(1)
 
 
-def run_build(litani, jobs, fail_on_proof_failure):
+def run_build(litani, jobs, fail_on_proof_failure, tabular_summary_print_fmt):
     cmd = [str(litani), "run-build"]
     if jobs:
         cmd.extend(["-j", str(jobs)])
     if fail_on_proof_failure:
         cmd.append("--fail-on-pipeline-failure")
+    if tabular_summary_print_fmt:
+        out_file_path = pathlib.Path("run.json")
```

Comment:
Why dump `run.json` in the `proofs` directory? It should at least be `.gitignore`d... but can I suggest a different location, like use `tempfile` to create a temporary directory to write into?